Paloma/contacto-html/paloma-contacto.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="css/mycss.css" />
+    <meta charset="utf-8">
+    <title> Home </title>
+
+</head>
+
+<body>
+
+    <h1> Bienvenidos a mi pagina de contacto!</h1>
+    <h2> Soy Paloma! Encantada de que esten aqui </h2>
+<img src="/contacto-html/assets/iJXsvqIa_400x400.jpg" height="40%" width="30%" alt="Paloma"/>
+    <p> Esta soy yo </p>
+
+    <p> Hola! Me llamo Paloma, vengo de México, estuve viviendo 20 años en <i>Dallas, Texas</i> aunque estuve ahi
+        viviendo bastante tiempo, no me gustó <b>NADA!</b>
+    <p>Llevo viviendo en España aproximadamente 6 años yendo y viniendo de aqui a Dallas para ver a mi familia que sigue
+        ahi</p>
+    <p>Me gusta mucho cocinar y más que cocinar, comer. Me ecanta viajar, y lo intento hacer cada vez que hay
+        oportunidad. Uno de los sitios que más me ha gustado ha sido Peru</p>
+    <p>El email para contactarme es: <a href="mailto:emailfalso@gmail.com"> este </a> </p>
+
+    <p><b>Ultimas 5 peliculas que vi:</b></p>
+    <ol>
+        <li>
+            <p>Circo de los Extraños</p>
+        </li>
+        <li>
+            <p>The Woman in the Window</p>
+        </li>
+        <li>
+            <p>Hotel Transilvania</p>
+        </li>
+        <li>
+            <p>The Silence of the Lambs</p>
+        </li>
+        <li>
+            <p>Shrek</p>
+        </li>
+
+    </ol>
+    <p><b>Series que estoy viendo ahora</b></p>
+    <ul>
+        <li>
+            <p>Orange is the new black</p>
+        </li>
+        <li>
+            <p>100 dias para enamorarse</p>
+        </li>
+        <li>
+            <p>South park</p>
+        </li>
+        <li>
+            <p>Family guy</p>
+        </li>
+        <li>
+            <p>The office</p>
+        </li>
+    </ul>
+
+   <p> Me encanta comer y disfrutar de la comida y cocinar...<img src="/contacto-html/assets/20210624_152818.jpg" height="40%" width="30%"alt="Fideua"/> </p>
+<p>Y este es el ultimo sitio al que fui! <img src="/contacto-html/assets/20210623_214110.jpg" height="20%" width="30%" alt="Tavernes"/></p>
+
+
+</body
```

Comment:
close body tag and html tag. The modern browser will render this, but not the older